lib/client-api/src/index.js

```diff
@@ -13,3 +15,12 @@ export {
   pathToId,
   splitPath,
 };
+
+export const getQueryParams = () => {
+  return qs.parse(document.location.search, { ignoreQueryPrefix: true });
+};
+
+export const getQueryParam = key => {
```

Comment:
Should we also use this API in the story store where we read the query?

Reply:
Yeah sounds good to me